contracts/staking/contracts/src/immutable/MixinStorage.sol

```diff
@@ -72,7 +76,9 @@ contract MixinStorage is
     // mapping from Pool Id to Epoch
     mapping (bytes32 => uint256) internal _cumulativeRewardsByPoolLastStored;
 
-    // registered 0x Exchange contracts
+    /// @dev Registered 0x Exchange contracts, capable of paying protocol fees.
+    /// @param 0 The address to check.
+    /// @return 0 Whether the address is a registered exchange.
```

Comment:
> /// @return 0 Whether
Why `0` is this required for doc gen?

Reply:
(If this corresponds to return parameter at index `0`, could be nice to just default to 0)

Reply:
Hmm. The syntax becomes ambiguous that way, because we can't determine whether the first word of the comment is the parameter name or just part of the comment.

Reply:
Oh I see, I misinterpreted the `0` as the index but this the name we give in the docs. Maybe it would read better if we just gave them names?
```
/// @param exchangeAddress The address to check.
/// @return isRegistered Whether the address is a registered exchange.
```
Not sure if this would have any unintended consequences though. I'm sure either is fine.

Reply:
Yeah it would be cool to give them names but they won't jive with the ABI or the wrappers where they're just indexed. I don't know if that matters. We also have to be mindful of parameter ordering with this approach.